src/templates/layouts/event/index.tsx

```diff
@@ -88,6 +88,7 @@ export const Event = ({
               className="event-detail-img vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4"
               {...image}
               imageStyle="7_2_medium_thumbnail"
+              loading="eager"
```

Comment:
This image is always above the fold. Previously it was lazy-loading, which caused a large content repaint late in page render. This forces the image to load immediately, which is actually better for page rendering.